trino/overlays/prod/root-ca-secret.enc.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
```

Comment:
It appears this and the stage root-ca secret are the same, so should we move this to bases?